_includes/head.html

```diff
@@ -1,39 +1,58 @@
+	<meta property="og:description" content="PyConf Hyderabad is an yearly tech conference about python, its projects and development." />
+	<meta property="og:url" content="https://pyconf.hydpy.org/2020/" />
+	<meta property="og:type" content="website" />
+	
+	<!-- meta twitter -->
+	<meta name="twitter:title" content="PyConf Hyderabad 2020" />
+	<meta name="twitter:description" content="PyConf Hyderabad is an yearly tech conference about python, its projects and development." />
+	<meta name="twitter:url" content="https://pyconf.hydpy.org/2020/"/>
+	<meta name="twitter:card" content="summary_large_image">
+	<meta name="twitter:site" content="@pyconfhyd">
+	<meta name="twitter:creator" content="@pyconfhyd">
+	
+	<link
+	href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
+	rel="stylesheet"
+	/>
+	<!-- Preload assets-->
+	<link
+	rel="preload"
+	href="{{ 'assets/css/main.css' | relative_url }}"
+	as="style"
+	/>
+	
+	<!-- main CSS -->
+	<link rel="stylesheet" href="{{ 'assets/css/main.css' | relative_url }}" />
+	
+	<!-- favicon -->
+	<link rel="apple-touch-icon" sizes="57x57" href="{{ site.url }}{{ site.baseurl }}/favicons/apple-icon-57x57.png">
+	<link rel="apple-touch-icon" sizes="60x60" href="{{ site.url }}{{ site.baseurl }}/favicons/apple-icon-60x60.png">
+	<link rel="apple-touch-icon" sizes="72x72" href="{{ site.url }}{{ site.baseurl }}/favicons//assets/images/favicons/apple-icon-72x72.png">
+	<link rel="apple-touch-icon" sizes="76x76" href="{{ site.url }}{{ site.baseurl }}/favicons//assets/images/favicons/apple-icon-76x76.png">
+	<link rel="apple-touch-icon" sizes="114x114" href="{{ site.url }}{{ site.baseurl }}/favicons//assets/images/favicons/apple-icon-114x114.png">
+	<link rel="apple-touch-icon" sizes="120x120" href="{{ site.url }}{{ site.baseurl }}/favicons/apple-icon-120x120.png">
+	<link rel="apple-touch-icon" sizes="144x144" href="{{ site.url }}{{ site.baseurl }}/favicons/apple-icon-144x144.png">
+	<link rel="apple-touch-icon" sizes="152x152" href="{{ site.url }}{{ site.baseurl }}/favicons/apple-icon-152x152.png">
+	<link rel="apple-touch-icon" sizes="180x180" href="{{ site.url }}{{ site.baseurl }}/favicons/apple-icon-180x180.png">
+	<link rel="icon" type="image/png" sizes="192x192"  href="{{ site.url }}{{ site.baseurl }}/favicons/android-icon-192x192.png">
+	<link rel="icon" type="image/png" sizes="32x32" href="{{ site.url }}{{ site.baseurl }}/favicons/favicon-32x32.png">
+	<link rel="icon" type="image/png" sizes="96x96" href="{{ site.url }}{{ site.baseurl }}/favicons/favicon-96x96.png">
+	<link rel="icon" type="image/png" sizes="16x16" href="{{ site.url }}{{ site.baseurl }}/favicons/favicon-16x16.png">
+	<link rel="manifest" href="{{ site.url }}{{ site.baseurl }}/favicons/manifest.json">
+	<meta name="msapplication-TileColor" content="#ffffff">
+	<meta name="msapplication-TileImage" content="{{ site.url }}{{ site.baseurl }}/favicons/ms-icon-144x144.png">
+	<meta name="theme-color" content="#ffffff">
```

Comment:
change theme-color to either orange or dark orange to match with the headings on page and theme.